pygromos/files/gromos_system/ff/serenityff/serenityff.py

```diff
@@ -24,7 +24,7 @@
 else:
     from openforcefield.topology import Molecule
     from openforcefield.typing.engines import smirnoff
-    from pygromos.files.gromos_system import openforcefield2gromos
+    from pygromos.files.gromos_system.ff import openforcefield2gromos
 
 
 class serenityff():
```

Comment:
LGTM complains here about line 40 

self.off = openforcefield2gromos(openFFmolecule=self.offmol, gromosTop=self.top, forcefield_name=forcefield)
NEWCall to a non-callable of builtin-class module.

Reply:
and fixed